src/test/java/com/blackducksoftware/integration/hub/alert/digest/DigestNotificationProcessorIT.java

```diff
@@ -138,7 +137,6 @@ public void processNotificationDataWithSameEventKeyTestIT() {
         assertEquals(1, eventsCreated.size());
         final AbstractChannelEvent event = eventsCreated.get(0);
         assertTrue(event instanceof HipChatEvent);
-        assertEquals(hipChatUserRel.getUserConfidId(), event.getUserConfigId());
```

Comment:
We cannot predict the id in the tests. With auto-increment and no guaranteed test order, it is very likely to change.